# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/rene/kdirstat/ogg-color.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- kdirstat-2.4.0/kdirstat/ktreemapview.cpp.orig	2003-12-24 01:26:01.720000000 +0100
+++ kdirstat-2.4.0/kdirstat/ktreemapview.cpp	2003-12-24 01:26:44.110000000 +0100
@@ -660,6 +660,7 @@
 
 		if ( lowerExt == "wav"	)	return Qt::yellow;
 		if ( lowerExt == "mp3"	)	return Qt::yellow;
+		if ( lowerExt == "ogg"  )	return Qt::yellow;
 
 		if ( lowerExt == "avi"	)	return QColor( 0xa0, 0xff, 0x00 );
 		if ( lowerExt == "mov"	)	return QColor( 0xa0, 0xff, 0x00 );
